modules/gitops-installing-argo-rollouts-cli-using-mac-os.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argo-rollouts-using-mac-os_{context}"]
+= Installing Argo Rollouts CLI using Mac OS
+
+For macOS, you can install the Argo Rollouts CLI by using the link:https://brew.sh[Homebrew] package manager.
+
+.Prerequisites
+
+* You must have Homebrew (`brew`) installed.
+
+.Procedure
+
+* Run the following command to install the Argo Rollouts CLI.
++
+[source,terminal]
+----
+$ brew install argoproj/tap/kubectl-argo-rollouts
```

Comment:
Any verification steps here? Or command outputs?

Reply:
As per the walkthrough demo provided by the SME, we did not view any additional output after this step.

Reply:
Correct, I based the structure of this section on https://docs.openshift.com/container-platform/4.15/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli-brew_cli-developer-commands